src/content/reference/cli.mdx

```diff
@@ -438,6 +438,32 @@ $ okteto namespace use cindy
  ✓  Using context cindy @ cloud.okteto.com
 ```
 
+#### sleep
+
+Sleeps the okteto namespace. 
+
+```console
+$ okteto namespace sleep test-cindy
+```
+
+```console
+i  Using test-cindy @ cloud.okteto.com as context
+✓  Namespace 'test-cindy' is sleeping
+```
+
+#### wake
+
+Wakes the sleeping okteto namespace
+
+```console
+$ okteto namespace wake test-cindy
+```
+
+```console
+i  Using test-cindy @ cloud.okteto.com as context
+✓  Namespace 'test-cindy' is awake now
```

Comment:
does the cli wai for the wake operation to finish?

Reply:
It waits for 1-2 sec.